frontend/components/layout/Header.vue

```diff
@@ -72,7 +75,7 @@ export default Vue.extend({
         type: {
             type: String,
             default: 'default',
-            validator: type => ['default', 'unified'].includes(type),
+            validator: type => ['default', 'unified', 'features'].includes(type),
```

Comment:
Can you reuse `unified` type or is it in some way different?

Reply:
using `minimal` from privacy page now 🙂